partner_contact_in_several_companies/__init__.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from . import models
+import models
```

Comment:
Please restore the `from .` on all `__init__.py` files.

Using `from . import models` is required, to avoid conflicts with a python module outside Odoo which could be called `models`. There is a lint check about this.